ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -2102,7 +2102,7 @@ signal_list_t* connect_function_instantiation_and_alias(short PASS, ast_node_t*
 
         // IF - this condition creates the circuitry of the expressions used in the function list
         if (i > 0 && (function_instance_var_node->type == BINARY_OPERATION || function_instance_var_node->type == UNARY_OPERATION || function_instance_var_node->type == TERNARY_OPERATION || function_instance_var_node->type == CONCATENATE)) {
-            input_signals = netlist_expand_ast_of_module(&(function_instance_var_node), instance_name_prefix, local_ref, port_size);
+            input_signals = netlist_expand_ast_of_module(&(function_instance_var_node), instance_name_prefix, local_ref, port_size, NO_ID);
```

Comment:
As a general review, these NO_IDs are not good and the reason for that is the `netlist_expand_ast_of_module` will call recursively, you do not know what will pass in each recursion. As a result, passing this may result in malfunction. Using the parent type each time would be much better.